src/main/java/org/breedinginsight/brapps/importer/model/imports/BrAPIImport.java

```diff
@@ -17,5 +17,16 @@
 
 package org.breedinginsight.brapps.importer.model.imports;
 
+import org.breedinginsight.brapps.importer.model.base.*;
+
 public interface BrAPIImport {
+
+    default Germplasm getGermplasm() { return null; }
+    default Trial getTrial() { return null; }
+    default Location getLocation() { return null; }
+    default Study getStudy() { return null; }
+    default ObservationUnit getObservationUnit() { return null; }
+    default Observation getObservation() { return null; }
+    default ObservationVariable getObservationVariable() { return null; }
```

Comment:
ya, previously there was casting being done in what would now be processor code that was coupling the import type and processor. I wanted to decouple those so that it didn't matter what type of import was coming into the processor, it would just look at objects it cared about in the import.